lib/auth/windows/ldap.go

```diff
@@ -22,6 +22,8 @@ import (
 	"crypto/x509"
 	"errors"
 	"fmt"
+	"github.com/go-asn1-ber/asn1-ber"
```

Comment:
Do we need this?

Reply:
Yes, it's used in `extractReferrals` (`packet *ber.Packet`)

Reply:
Does Go's `encoding/asn1` package work instead?

Reply:
For my curiosity: is it answered somewhere?